vendor/brave-ios/Ads/BATBraveAds.mm

```diff
@@ -396,6 +415,16 @@ - (void)getCreativeAdNotifications:(const std::vector<std::string> &)categories
   callback(ads::Result::SUCCESS, categories, found_ads);
 }
 
+- (void)getCreativePublisherAds:(const std::string &)url categories:(const std::vector<std::string> &)categories sizes:(const std::vector<std::string> &)sizes callback:(ads::OnGetCreativePublisherAdsCallback)callback
+{
+  // TODO(brave): To be implemented
```

Comment:
What needs to be implemented here?

Reply:
Updated comment. However, this work is not required until Publisher Ads are required on iOS